packages/clarity-js/types/data.d.ts

```diff
@@ -115,7 +115,8 @@ export const enum Code {
     PerformanceObserver = 3,
     CallStackDepth = 4,
     Selector = 5,
-    Metric = 6
+    Metric = 6,
+    CSP = 7
```

Comment:
Nit: For acronyms what's the casing rule? Is it CSPRule or CspRule? If former this is fine, if latter should be Csp.

Reply:
Changed it to ContentSecurityPolicy for now.